docs/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Extend CVSS vector length (OSIDB-3362)
+- Sync trackers on impact decrease (OSIDB-3350)
```

Comment:
In case there is no conflict raised by github, rebase changelog as 4.2.0 was already released.